src/ui.py

```diff
@@ -0,0 +1,291 @@
+                                     height=100 if height == 1 else 100*height,
+                                     hover=True,
+                                     command=command_dict[json_data["useState"]])
+
+        button.grid(row=location["start"]["row"],
+                    column=location["start"]["col"], rowspan=height, columnspan=width)
+
+
+class InformationFrame(tk.Frame):
+    """
+    사물함을 클릭했을 때 정보를 입력할 프레임입니다.
+    """
+
+    def __init__(self, parent, controller, *args, **kwargs):
+        super().__init__(parent, *args, **kwargs)
+        self.controller = controller
+        intro_label = tk.Label(
+            self, text="휴대폰 번호를 입력해주세요.", font=controller.large_font)
+        entry = tk.Entry(self)
+        number_frame = tk.Frame(self)
+        before_button = TkinterCustomButton(master=self,
+                                            bg_color=None,
+                                            fg_color="#2874A6",
+                                            border_color=None,
+                                            hover_color="#5499C7",
+                                            text_font=None,
+                                            text="이전으로",
+                                            text_color="white",
+                                            corner_radius=10,
+                                            border_width=1,
+                                            width=100,
+                                            height=100,
+                                            hover=True,
+                                            command=lambda: UIEvent.show_error(
+                                                message="이전으로 돌아가기")
+                                            )
+        row = 0
+        col = 0
+        # ===================================
+        button_name_list = ["1", "2", "3", "4", "5",
+                            "6", "7", "8", "9", "취소", "0", "확인"]
+        for i in button_name_list:
+            text = f"{i} 버튼 이벤트 발생"
+            button = TkinterCustomButton(master=number_frame,
+                                         bg_color=None,
+                                         fg_color="#2874A6",
+                                         border_color=None,
+                                         hover_color="#5499C7",
+                                         text_font=None,
+                                         text=i,
+                                         text_color="white",
+                                         corner_radius=10,
+                                         border_width=1,
+                                         width=100,
+                                         height=100,
+                                         hover=True,
+                                         command=lambda: UIEvent.show_error(
+                                             message=text)
```

Comment:
```suggestion
                                         command=lambda text=text: UIEvent.show_error(
                                             message=text)
```
또는
```suggestion
                                         command=lambda i=i: UIEvent.show_error(
                                             message=f"{i} 버튼 이벤트 발생")
```
로 해야 버튼이벤트 해결 가능할 듯.

Reply:
땡큐 감사 ❗️😭